package.json

```diff
@@ -13,7 +13,9 @@
     "url": "https://github.com/DevExpress/DevExtreme.AspNet.Data.git"
   },
   "main": "js/dx.aspnet.data.js",
+  "types": "js/dx.aspnet.data.d.ts",
   "dependencies": {
+    "@types/jquery": "^2.0.39",
```

Comment:
Not sure that this is correct. Not all npm users are using typescript. This dependency is listed in devDependencies of `devextreme` package for same reasons.

I suggest to move it in devDependencies section.